### ISSUE TITLE
No folders and subfolders in web navigation

### ISSUE THREAD

**Issue by Eldenroot** [prusa3d/Prusa-Link-Web#290]
As title says, there are only files placed in root displayed, no way to get the subfolders etc.

**Comment by ondratu** [comment]
Hi, please, can you tell me on which printer do you see that? You probably use MINI, which didn't has this functionality yet.

**Comment by Eldenroot** [comment]
Mini+ with Prusalink and 4.4.0RC fw

**Comment by afonsosantos** [comment]
Same behaviour here.
I would be great to see my subfolders on the web interface also!

**Comment by ondratu** [comment]
Yes, this functionality is not implemented here yet. It will be available in next release.
